src/Resources/config/messenger.xml

```diff
@@ -12,5 +12,9 @@
 
             <tag name="fos_elastica.pager_persister" persisterName="async" />
         </service>
+
+        <service id="fos_elastica.async_parsist_page_handler" class="FOS\ElasticaBundle\Message\Handler\AsyncPersistPageHandler">
+            <argument type="service" id="fos_elastica.async_pager_persister" />
```

Comment:
what about adding the tag here and register an alias for the command bus, that would remove the need of a compiler pass?

Reply:
See https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1744 as I'm not happy with my explanation.
It should now be just a story of tagging the service, and using the bus alias ;)